diff --git a/update.sh b/update.sh
index 6d844e8..b01bddb 100644
--- a/update.sh
+++ b/update.sh
@@ -45,12 +45,12 @@ downloadFromMicroG() {
        mkdir -p tmp
        if [ ! -f tmp/indexug.xml ];then
                #TODO: Check security keys
-               wget --connect-timeout=10 $repofd/index.jar -O tmp/indexug.jar
+               wget --connect-timeout=10 $repoug/index.jar -O tmp/indexug.jar
                unzip -p tmp/indexug.jar index.xml > tmp/indexug.xml
        fi
        marketvercode="$(xmlstarlet sel -t -m '//application[id="'"$1"'"]' -v ./marketvercode tmp/indexug.xml || true)"
        apk="$(xmlstarlet sel -t -m '//application[id="'"$1"'"]/package[versioncode="'"$marketvercode"'"]' -v ./apkname tmp/indexug.xml || xmlstarlet sel -t -m '//application[id="'"$1"'"]/package[1]' -v ./apkname tmp/indexug.xml)"
-       while ! wget --connect-timeout=10 $repofd/$apk -O bin/$apk;do sleep 1;done
+       while ! wget --connect-timeout=10 $repoug/$apk -O bin/$apk;do sleep 1;done
        addCopy $apk $1 "$2"
 }

@@ -58,7 +58,7 @@ downloadFromMicroG com.google.android.gms
 downloadFromMicroG com.google.android.gsf
 downloadFromMicroG com.android.vending
 downloadFromMicroG org.microg.gms.droidguard
-downloadFromMicroG org.microg.unifiednlp
+#downloadFromMicroG org.microg.unifiednlp

 #phh's Superuser
 downloadFromFdroid me.phh.superuser
